src/com/palmergames/bukkit/towny/object/Translation.java

```diff
@@ -63,6 +63,9 @@ public static void loadTranslationRegistry() {
 		translations = loader.getTranslations();
 		// Set the defaultLocale.
 		setDefaultLocale();
+		
+		// Remove any disabled languages from the translations map.
+		translations.keySet().removeIf(lang -> !TownySettings.isLanguageEnabled(lang) && !lang.equalsIgnoreCase(defaultLocale.toString()));
```

Comment:
I'm wondering if this shouldn't be in the TranslationLoader.

With it positioned here, other plugins will still be able to inject their strings in languages Towny would otherwise ignore.